schema-loader/src/main/java/com/scalar/db/schemaloader/command/SchemaLoaderCommand.java

```diff
@@ -100,18 +97,17 @@ public Integer call() throws Exception {
       metaOptions.put(DynamoAdmin.NO_BACKUP, noBackup.toString());
     }
 
-    SchemaOperator operator = SchemaOperatorFactory.getSchemaOperator(configPath);
+    SchemaOperator operator = SchemaOperatorFactory.getSchemaOperator(configPath.toString(), true);
 
     if (coordinator) {
       operator.createCoordinatorTable(metaOptions);
     }
 
     if (schemaFile != null) {
-      List<Table> tableList = SchemaParser.parse(schemaFile.toString(), metaOptions);
       if (deleteTables) {
-        operator.deleteTables(tableList);
+        operator.deleteTables(schemaFile, metaOptions);
       } else {
-        operator.createTables(tableList, metaOptions);
+        operator.createTables(schemaFile, metaOptions);
```

Comment:
Maybe we can change here to use `SchemaLoader.load()`?

Reply:
Since we have a case that only creating/deleting the coordinator table here, so I think we should keep using the operator.

Reply:
Sorry I didn't get it. What did you mean?

I thought we can do something like the following here:
```java
    if (!deleteTables) {
      SchemaLoader.load(configPath, schemaFile, metaOptions, coordinator);
    } else {
      SchemaLoader.unload(configPath, schemaFile, metaOptions, coordinator);
    }
```
What do you think?

Reply:
We can do like that, but we have a case that creating/deleting coordinator table without providing the schema file, then we need to use the operator or we have to add the methods for creating/deleting coordinator table to the `SchemaLoader` or handle the case passing a null `schemaFile` to load/unload method

Reply:
Okay understood. So I think `SchemaLoader.load()`/`SchemaLoader.unload()` should be able to handle that case (`schemaFile == null`). Maybe we can make the schemaFile parameter nullable (`@Nullable`) in `SchemaLoader.load()`/`SchemaLoader.unload()` or adding overloaded methods without the schemaFile parameter?

I think `SchemaLoader.load()`/`SchemaLoader.unload()` should have the full features of the schema loader.

Reply:
I make the methods to accept a null value of schemaFile. Applied in [commit](https://github.com/scalar-labs/scalardb/pull/366/commits/542830245f1e97ab1b3be0f05e4f8e2fe11342a5)